### DISCUSSION TITLE
Dokumentation openjverein

### DISCUSSION THREAD

**Discussion by dippeal** [openjverein/jverein#79] [General]
Hallo Leute,

ich möchte grundsätzlich abstimmen, wie bei diesem Projekt die Dokumentation hinterlegt werden soll.
Das Thema ist bei https://github.com/openjverein/jverein/pull/67#issuecomment-1808289829 aufgekommen.

Ich würde das [Gitbook](https://openjverein.gitbook.io/) mal als gesetzt annehmen. Dann kann man 

1. Einen PR erstellen vom persönlichen Fork des Gitbooks
2. Einen Change request im [Gitbook Repository](https://github.com/openjverein/jverein-Book) erstellen

Welches Vorhaben würdet ihr präferieren und warum?

**Comment by MSchmalzl** [comment]
Ich habe mich bei meiner Änderung für Methode 1 entschieden. Habe also einen PR vom persönlichen Fork des Gitbooks eingereicht.

Bisher ist es so, dass kein Review notwendig ist. Ich konnte also meine Änderung direkt selbst mergen.

**Comment by MSchmalzl** [comment]
Ich fand die Idee mit Gitbook am Anfang echt gut. Bis auf die schlechte erreichbarkeit der Server hat das auch funktioniert.

Bis ich dann vor ein paar Wochen eine Änderung einreichen wollte und hier gar nichts ging. Letztendlich habe ich mich dafür entschieden Gibook den rücken zu kehren.

Funktioniert Gitbook bei euch oder sollten wir uns nach einer Alternative umsehen?

**Comment by dippeal** [comment]
Also ich habe meine PR immer selbst genehmigt und n paar Sekunden später war die Anpassung online. Ging also ganz gut. Aussetzer hatte ich noch keine.

**Comment by MSchmalzl** [comment]
Hast du die kostenlose Version oder bezahlst du?

**Reply by dippeal** [reply]
Die kostenlose.
